docs/api_call.md

```diff
@@ -1,1352 +1,1154 @@
+        "detection": detection_method,
+        "original_text": original_text
+    },
+]
+```
 
-      ```json
-      {
-        "data": [
-          {
-            "detection": "structure_value_verified",
-            "original_text": "inferno",
-            "entity_value": {
-              "value": "Inferno"
-            }
-          }
-        ]
-      }
-      ```
+Consider the following example for detailed explanation:
 
-  - ​
+```
+"I want to order two litre pepsi."
+```
 
+- entity_value: This will store the value of entity (i.e entity value) that is detected. The output will be in dictionary format. For example, `{"value": "2", "unit": "quantity"}`.
+- detection: This will store how the entity is detected i.e. whether from *message*, *structured value* or *fallback value*.
+- original_text: This will store the actual value that is detected. For example, `two`.
 
-### phone_number
+```json
+Example 1:
+input:  message = 'I want to order 2 burgers from mainland china at 3 pm '
+        entity_name = 'restaurant'
+        structured_value = None
+        structured_value_verification = 0
+        fallback_value = None
+        bot_message = None
 
-- This functionality calls the PhoneDetector class to detect textual entities.
+output:
+[
+  {
+    "detection": "message",
+    "original_text": "2",
+    "entity_value": { "value": "2", "unit": null},
+    "language": "en"
+  },
+  {
+    "detection": "message",
+    "original_text": "3 pm",
+    "entity_value": { "mm": 0, "hh": 3, "nn": "pm"},
+    "language": "en"
+  }
+]
 
-- Example:
+Example 2:
+// referenced date is 19th feb
+input:  message = 'I have my maths exam next Saturday.'
+        entity_name = 'date'
+        structured_value = None
+        structured_value_verification = 0
+        fallback_value = None
+        bot_message = None
 
-  - Example 1: 
+output:[
+{
+    "detection": "message",
+    "original_text": "inferno",
+    "entity_value": {"value": {"mm": 03, "yy": 2019, "dd": 02, "type": "date"}
+  }, 
+]
+```
 
-    - ```python
-      message = 'my contact number is 9049961794'
-      entity_name = 'phone_number'
-      structured_value = None
-      fallback_value = None
-      bot_message = None
-      ```
+###  
 
-    - *Python:* 
+### Installing Chatbot NER:
 
-      ```python
-      from ner_v1.chatbot.entity_detection import get_phone_number
-      output = get_phone_number(message=message, entity_name=entity_name, structured_value=structured_value, fallback_value=fallback_value, bot_message=bot_message)
-      print output
-      ```
+Please, have a look at [installation steps](https://github.com/hellohaptik/chatbot_ner/blob/master/docs/install.md) guide to install Chatbot NER on your system.
 
-    - *CURL command:*
+After following above steps, you can test the APIs either using django shell or by making curl get request. To get into django shell, you need to run `python manage.py shell` command in main project directory. 
```

Comment:
Since installation only supports Docker, how to get into the container should be added before instructing the users to run `python manage.py shell`